### PR TITLE
add module for shigeifinder

### PR DESCRIPTION
**Pull request by rpetit3**
## PR checklist

Closes https://github.com/nf-core/modules/issues/2522 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/shigeifinder/meta.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - seqs:
+      type: file
+      description: Assembly or paired-end Illumina reads
+      pattern: "*.{fasta.gz,fasta.gz}"
```
Comment (SPPearce) [inline modules/nf-core/shigeifinder/meta.yml:26]: Does it have to be gzipped or would it work with unzipped data? Should the second one be `fastq.gz`

Comment (rpetit3) [reply modules/nf-core/shigeifinder/meta.yml]: good catch, updated

**Review by SPPearce — APPROVED** [review]
Looks good to go then.

**Comment by rpetit3** [comment]
Thank you!

**Pull request merged**